kinematics_base_test/src/test_kinematics_plugin.cpp

```diff
@@ -109,6 +111,14 @@ class KinematicsTest {
   };
 
 public:
+  // Definitions for handling smart pointers in Kinetic
```

Comment:
I am not sure about it. Maybe @jrgnicho can weigh in here. In the [latest commit(https://github.com/ros-planning/moveit_kinematics_tests/pull/13), we actually don't need these definitions. These defintions were meant to handle [this](https://github.com/ros-planning/moveit_kinematics_tests/blob/734104340f01d8db9d9690055c5e23e1b52886b1/kinematics_base_test/src/test_kinematics_plugin.cpp#L46). Will remove these in next commit.